scripts/check-qemu-install

```diff
@@ -162,13 +155,10 @@ def get_qemu_cmd(name, enable_uefi, disk_img, raid=None, iso_img=None, tpm=False
                 f' -device ahci,id=achi0' \
                 f' -device ide-cd,bus=achi0.0,drive=drive-cd1,id=cd1,bootindex=10'
 
-    # test using half of the available CPUs on the system
-    cpucount = get_half_cpus()
-
-    macbase = '52:54:00:00:00'
+    macbase = '02:00:00:00:00'
     cmd = f'qemu-system-x86_64 \
         -name "{name}" \
-        -smp {cpucount},sockets=1,cores={cpucount},threads=1 \
+        -smp 2,sockets=1,cores=2,threads=1 \
```

Comment:
Why only two cores? Are there any issues with more cores?

Reply:
No. It's a sane and good default.

I've encountered issues with more cores (auto detected previously) when the host system has Intel P and E cores.